core/controllers/feedback.py

```diff
@@ -58,7 +59,12 @@ class ThreadHandler(base.BaseHandler):
 
     def get(self, exploration_id, thread_id):
         self.values.update({
-            'messages': feedback_services.get_messages(thread_id)})
+            'messages': feedback_services.get_messages(".".join((
```

Comment:
nit, here and below: single quotes in python

Would it be hard to change feedback_services.get_messages to take two args: exploration_id and thread_id? I'm concerned about having two definitions of thread_id; it would be good to clean this up.